### DISCUSSION TITLE
Copernicus Sentinel-2: Major Products Upgrade Upcoming

### DISCUSSION THREAD

**Discussion by vincentschut** [davidfrantz/force#125] [Q&A] [answered]
According to https://scihub.copernicus.eu/news/News00931, several things will change in the S2 products as delivered by ESA, per October 26. Just asking here: any idea whether FORCE as it is now will still be compatible with these?

**Comment by davidfrantz** [comment]
Thanks for the notification @vincentschut 
I assume that the addition of the additive scaling factor might need some adaptation. I will look into this once the sample data will become available.
Cheers, David

**Comment by vincentschut** [comment]
Fyi, sample products are here: https://scihub.copernicus.eu/news/News00935
thanks for looking into this!

**Comment by davidfrantz** [comment, marked as answer]
Hi Vincent,

as suspected, the baseline 4.0 change did not work with FORCE. Please see the issue based on your input:
https://github.com/davidfrantz/force/issues/132

I released a new version that should fix this issue: **https://github.com/davidfrantz/force/releases/tag/v3.7.2**

Please update your processes accordingly.

Cheers and thanks again for the heads-up,
David

**Comment by vincentschut** [comment]
Hi David,

thanks for fixing this so quick! We will make sure our processing pipelines use FORCE >= 3.7.2 before the baseline change takes place.

Vincent.
